docs/emissions.md

```diff
@@ -1,169 +1,111 @@
+### Injection
 
-## Before you proceed
+The first stage of emissions is *injection of liquidity* into the subnet pools. Each block:
 
-Read the [Root Network](./root-network.md) document before you proceed.
+- TAO is injected into the subnet's TAO reserve.
+- alpha is injected into the subnet's alpha reserve.
+- alpha is allocated to *alpha outstanding*, to be extracted by participants.
 
-In the rest of this document we consider the subnet weights (set by the root validators) as **inputs** and proceed to present emission calculations as **outputs**. 
+Liquidity injection to each subnet is in proportion to the price of its token compared to the price other subnet tokens. This is designed to incentivize development on the most valuable subnets. Recall that price is equal to the ratio of the subnet's TAO in reserve to alpha in reserve.
 
-## Subnet weights, trust, rank, consensus, emission 
+#### TAO reserve injection
 
-### Read root network metagraph
-
-Consider a snapshot of root network at a given block. You can read the root network metagraph with the below `metagraph` call:
-
-```python
-import torch
-metagraph = bt.metagraph(netuid=0, lite=False)
-metagraph.weights.shape
-```
-
-Running the above code will give you the shape of the weight matrix of the root network. For example, when total number of subnets are 32 (there is no limit to the number of subnets):
-
-```python
-torch.Size([64, 33])
-```
-
-As expected, the shape of the weights reflects the 64 root network validators that set the weights for the example case of 32 subnets. The root network itself is counted, hence `33` in the above output, instead of the example number 32.
-
-You can then read the weights matrix $W$ from the metagraph as below. See the [`metagraph` API documentation](https://docs.bittensor.com/python-api/html/autoapi/bittensor/metagraph/index.html#bittensor.metagraph.metagraph.W).
-
-```python
-# Create a weight matrix with FP32 resolution
-W = metagraph.W.float()
-```
-
-Next, read the stake vector $S$. See [`metagraph` property `S` documentation](https://docs.bittensor.com/python-api/html/autoapi/bittensor/metagraph/index.html#bittensor.metagraph.metagraph.S).
-
-```python
-# Create "normalized" stake vector
-Sn = (metagraph.S/metagraph.S.sum()).clone().float()
-```
-
-Next, we describe how to compute the below quantities that are defined on a subnet.  
-
-- **Trust** ($T$)
-- **Consensus** ($C$)
-- **Rank** ($R$)
-- **Emission** ($E$)
-
-### Trust
-
-Trust is defined as a sum of only those stakes that set non-zero weights. 
+Given set S of all subnets, and a total per block TAO emission $\Delta\bar{\tau}$, which begins at 1 TAO and follows a halving schedule, TAO emission $\Delta\tau_i$ to subnet $i$ with price $p_i$ is:
```

Comment:
should we mention that it's an EMA?

Reply:
yes, thanks, could you remind me what the window is for the EMA?